core/lib/generators/solidus/install/install_generator/install_frontend.rb

```diff
@@ -11,23 +11,31 @@ def initialize(bundler_context:, generator_context:)
         @generator_context = generator_context
       end
 
-      def call(frontend, installer_adds_auth:)
+      def call(frontend)
         case frontend
         when 'solidus_frontend'
           install_solidus_frontend
         when 'solidus_starter_frontend'
-          install_solidus_starter_frontend(installer_adds_auth)
+          install_solidus_starter_frontend
         end
       end
 
       private
 
+      def bundle_command(command, env = {})
```

Comment:
Not sure I understand the need for this change. In which situation is <code>Kernel#`</code> incompatible? It looks like we're adding unneeded complexity, but I'm sure I'm missing something.

Reply:
The incompatibility comes down to Kernel#` sending the whole command to the current shell (which is not guaranteed to be `bash` on windows). The reason for this helper is surely questionable, but I went straight to the original Rails implementation instead of trying to find it out on my own. That's where I got that we could use `Gem.ruby` etc. for composing the command to be sent to system. I did that of course because I know Rails works on windows and it seemed the safest bet.

Reply:
Fair enough! Let's go with it!